sc2ts/nextstrain.py

```diff
@@ -162,3 +164,93 @@ def get_mutation_path(ts, node):
                     (int(site.position), inherited_state, mut.derived_state)
                 )
     return depth_map
+
+
+
+def newick_from_nextstrain_with_comments(
+    tree_string, min_edge_length, time_units, convert_to_days=True):
```

Comment:
Why do we have ``time_units`` and ``convert_to_days`` as arguments - don't these interact? I would say just convert to days-ago from the most recent sample like we have in the sc2ts trees. The num_date values they provide are a bit irritating to work with all right though.

Reply:
As in, don't provide any options.

Reply:
Yes, I wasn't sure if *all* the onewicks they provide have times in years, but I could just assume that.

In the plots, I think we should express all the days in terms of time since e.g. 1st Jan 2020 (or maybe the date of the first Wuhan sample), rather than days-ago, as then we have comparable scales even if we add more samples into the sc2ts tree. But we can keep the ts dates in days-ago and simply convert in the plots, which is what we have done.

Reply:
Just blindly converted to years, with an error check that the times span less then 20 units, which should warn us if we accidentally use a file with days.